scripts/userstyles.yml

```
@@ -519,7 +521,15 @@ userstyles:
     color: blue
     readme:
       app-link: "https://www.linkedin.com"
-    current-maintainers: [*isabelroses]
+      current-maintainers: [*isabelroses]
+  listenbrainz:
+    name: ListenBrainz
+    categories: [music, entertainment]
+    icon: musicbrainz
+    color: maroon
+    readme:
+      app-link: 'https://listenbrainz.org'
+      current-maintainers: [*00dani]
```

We have recently hoisted this value up.
```suggestion
  listenbrainz:
    name: ListenBrainz
    categories: [music, entertainment]
    icon: musicbrainz
    color: maroon
    readme:
      app-link: 'https://listenbrainz.org'
    current-maintainers: [*00dani]
```